charts/manager/templates/manager-role.yaml

```diff
@@ -1,7 +1,5 @@
-{{/* 
-SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
-SPDX-License-Identifier: Apache-2.0
-*/}}
```

Comment:
make generate does not yet add license headers and the GH action does not yet add the correct comment style. Could you revert to the `{{/* */}}` syntax for SPDX comments?

Reply:
Do we need the license header in the helm charts then?

Reply:
done via [e97fd86](https://github.com/cloudoperators/greenhouse/pull/75/commits/e97fd862a9139e9233c3e4faf39db5ea018afd42)